equivalences.csv

```diff
@@ -20,6 +20,7 @@ BEL,ALDO Family,ALDO
 BEL,AMP Activated Protein Kinase Complex,AMPK
 BEL,Annexin II Heterotetramer Complex,Annexin_II_heterotetramer
 BEL,AP-1 Complex,AP1
+BEL,AP-2 complex,Adaptor_protein_II

```

Comment:
Should this be capitalized to "AP-2 complex", as with the "AP-1 Complex"?

Reply:
I've fixed a few things but can't update this PR--can you allow edits by maintainers? See: https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/